src/content-routing.js

```diff
@@ -24,7 +29,7 @@ module.exports = (node) => {
      * @param {object} [options]
      * @param {number} [options.timeout] - How long the query should run
      * @param {number} [options.maxNumProviders] - maximum number of providers to find
-     * @returns {AsyncIterable<{ id: PeerId, multiaddrs: Multiaddr[] }>}
+     * @returns {AsyncIterable<{ id: PeerId, multiaddrs: Array<multiaddr> }>}
```

Comment:
I believe `T[]` is more idiomatic style that `Array<T>`, only cases where later is recommended over the former are when otherwise it's ambiguous e.g: 

```
[T, U][] -> Array<[T, U]>
T[][] -> Array<T[]>
```

Reply:
We are doing the `Array<>` everywhere in libp2p. But, I think we can change to `T[]` as it is also faster to type.